.github/workflows/Packages - EdFi.Ods.CodeGen.yml

```diff
@@ -21,49 +21,48 @@ env:
 
 jobs:
   build:
-
-    runs-on: windows-latest
-
+    runs-on: ubuntu-latest
```

Comment:
I don't see anything in this script that would require Windows instead of Ubuntu. Note: In the Admin API project, we publish NuGet packages to Azure Artifacts using Ubuntu.